core/templates/dev/head/pages/skill-editor-page/navbar/skill-editor-navbar.directive.html

```diff
@@ -86,7 +85,14 @@
   </li>
 </ul>
 <style>
-  skill-editor-navbar .save-draft-button-padding {
-    padding-right: 120px;
+  .btn-success:hover {
+    background-color: #218838 ;
+    border-color: #1e7e34 ;
+    color: #fff;
+  }
+  .btn-success {
+    background-color:  #5cb85c; ;
```

Comment:
The extra space and the double semicolons suggest a lack of proofreading.

Reply:
I've corrected this and looking for more such issues